.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "PaddleNLP/PALM"]
 	path = PaddleNLP/PALM
 	url = https://github.com/PaddlePaddle/PALM
+[submodule "PaddleCV/Paddle3D/PointRCNN/pybind11"]
+	path = PaddleCV/Paddle3D/PointRCNN/pybind11
+	url = https://github.com/pybind/pybind11
```

Comment:
Maybe better to tell users  to git clone in doc, not as submodule?

Reply:
Done, Thanks!